packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -485,10 +499,11 @@ export function createStrictDOMComponent<T, P: StrictProps>(
         // Add default text styles
         nativeComponent === Text && styles.userSelectAuto,
         // Provided styles
-        nativeComponent === Text || nativeComponent === TextInput
+        nativeComponent === Text
```

Comment:
`TextInput` was incorrectly inheriting text styles, but this is not how `input` behaves on web.